Tests/NostrSDKTests/RelayResponseDecodingTests.swift

```diff
@@ -99,23 +105,52 @@ final class RelayResponseDecodingTests: XCTestCase, FixtureLoading {
             }
             XCTAssertEqual(eventId, "b1a649ebe8b435ec71d3784793f3bbf4b93e64e17568a741aecd4c7ddeafce30")
             XCTAssertEqual(success, false)
-            XCTAssertEqual(message.type, .blocked)
+            XCTAssertEqual(message.prefix, .blocked)
             XCTAssertEqual(message.message, "tor exit nodes not allowed")
         } else {
             XCTFail("failed to decode")
         }
     }
 
-    func testDecodeCount() throws {
-        let data = try loadFixtureData("count_response")
+    func testDecodeEOSEMessage() throws {
+        let data = try loadFixtureData("eose")
 
         if let relayResponse = RelayResponse.decode(data: data) {
-            guard case .count(let subscriptionId, let count) = relayResponse else {
+            guard case .eose(let subscriptionId) = relayResponse else {
                 XCTFail("incorrect type")
                 return
             }
-            XCTAssertEqual(subscriptionId, "subscription-id")
-            XCTAssertEqual(count, 238)
+            XCTAssertEqual(subscriptionId, "some-subscription-id")
+        } else {
+            XCTFail("failed to decode")
+        }
+    }
+
+    func testDecodeClosedMessage() throws {
+        let data = try loadFixtureData("closed")
+
+        if let relayResponse = RelayResponse.decode(data: data) {
```

Comment:
Suggestion: I see that this was the existing pattern in this file, and keeping that is fine. In general, however, it is cleaner to use `XCTUnwrap` rather than `if let`/`guard let` in tests.

Reply:
I cleaned up all those usages in this test file. Thanks!